### DISCUSSION TITLE
"Files" app

### DISCUSSION THREAD

**Discussion by koersner** [community/community#48907] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello. I am new to github so maybe I'm doing something wrong here. But I have followed the development of the Files app for windows with great interest and it looks very promising. Unfortunately, on my Windows 11 system which I consider to be ok it crashes after a couple of seconds. Is this well known ? I have tried the last "stable" version 2.4.40 from Microsoft store or the latest from github which is - also 2.4.40. 
If i posted this in the wrong place just let me know. I'm just so eager to get Files running.
Regards
Ib Kôersner
Sweden

**Comment by marcofranzen99** [comment]
Hi @koersner , i think this is not quite the right place for feedback regarding the Files app. I think this area is more for general discussions.

If you want to create a bug or feature report for the Files app, you can do that directly in the Files repository: https://github.com/files-community/Files/issues

Alternatively you can join the Discord server. There is a general chat, but also one where you can get help if you have problems with the app.
https://discord.gg/files

**Comment by koersner** [comment]
Thanks a lot. It runs very well now.
Ib



------ Original Message ------
From "Marco Franzen" ***@***.***>
To "community/community" ***@***.***>
Cc "koersner" ***@***.***>; "Author" 
***@***.***>
Date 29-Mar-23 13:04:21
Subject Re: [community/community] "Files" app (Discussion #48907)
